.github/workflows/ci-python-sdk.yaml

```diff
@@ -1,21 +1,22 @@
-name: Build and test astro
+name: Build and test astro Python SDK
 on:
   push:
     branches: [ 'main', 'release-**' ]
     paths:
       - 'python-sdk/**'
       - '.github/workflows/ci-python-sdk.yaml'
-      - '*'
   pull_request:
     branches: [ 'main', 'release-**' ]
     paths:
       - 'python-sdk/**'
       - '.github/workflows/ci-python-sdk.yaml'
-      - '*'
```

Comment:
We don't want to trigger `python-sdk` tests if only the `sql-cli` changes

Reply:
The `*` here only finds files in the root directory e.g. `.pre-commit-config.yaml`